core/silkworm/common/util.cpp

```diff
@@ -123,17 +123,11 @@ evmc::bytes32 to_bytes32(ByteView bytes) {
     return out;
 }
 
-ByteView zeroless_view(const ByteView& data) {
+ByteView zeroless_view(ByteView data) {
```

Comment:
data is not consumed nor altered.
I'd have it passed by const ref

Reply:
`ByteView` is a small type. Google also suggests to pass it by value: see "A Few Additional Notes" in https://abseil.io/tips/1.

Reply:
I understand that but my point is made up of :
- even if small is neverheless the copy of a pointer **and** a size
- google says "should" ... not "must"
- google says we can keep consistency with surrounding code
Hence, imho, `const sv& data` does no harm and gives immediate picture the param is not consumed (which is consistent with other functions)